app/src/main/java/com/onewelcome/showcaseapp/feature/userregistration/browserregistration/BrowserRegistrationScreen.kt

```diff
@@ -156,15 +157,17 @@ private fun IdentityProvidersSection(
   onEvent: (UiEvent) -> Unit
 ) {
   if (uiState.identityProviders.isNotEmpty()) {
-    Column {
-      IdentityProvidersHeader()
-      IdentityProvidersList(uiState.selectedIdentityProvider, uiState.identityProviders, onEvent)
-      ShowcaseSwitch(
-        shouldBeChecked = uiState.shouldUseDefaultIdentityProvider,
-        onCheck = { onEvent.invoke(UiEvent.UseDefaultIdentityProvider(it)) },
-        text = stringResource(R.string.use_default_identity_provider),
-        tooltipContent = { Text(stringResource(R.string.default_identity_provider_tooltip_text)) }
-      )
+    ShowcaseCard {
+      Column {
+        IdentityProvidersHeader()
+        IdentityProvidersList(uiState.selectedIdentityProvider, uiState.identityProviders, onEvent)
+        ShowcaseSwitch(
+          shouldBeChecked = uiState.shouldUseDefaultIdentityProvider,
+          onCheck = { onEvent.invoke(UiEvent.UseDefaultIdentityProvider(it)) },
+          text = stringResource(R.string.use_default_identity_provider),
+          tooltipContent = { Text(stringResource(R.string.default_identity_provider_tooltip_text)) }
+        )
+      }
     }
   }
 }
```

Comment:
As you're already commiting to this file, could I ask you to also wrap `IdentityProvidersList` in the `ShowcaseCard`?

Reply:
hmm, I added that in the method above 🤔 This way it wraps in card the identity providers list toghether with labels and switch

Reply:
Sorry 😓 I meant registration scopes!

![image](https://github.com/user-attachments/assets/1af5629f-e8f1-4bc6-8d7b-8b1c734e1340)